metal3-dev/reset-hosts.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+bmo_path=$GOPATH/src/github.com/metal3-io/baremetal-operator
+if [ ! -d $bmo_path ]; then
+    echo "Did not find $bmo_path" 1>&2
+    exit 1
+fi
+
+hosts=$(oc get baremetalhost -n openshift-machine-api -o name | cut -f2 -d/)
+
+(pushd $bmo_path &&
+     go run ./cmd/reset-host/main.go -n openshift-machine-api $hosts)
+
+# NOTE: assumes OS_CLOUD is set correctly
+for host in $hosts; do
+    openstack baremetal node maintenance set $host || continue
+    openstack baremetal node delete $host
```

Comment:
This will only work when the script is run from the dev-scripts root directory, we could make it also work when folks `cd metal3-dev` if we define `SCRIPTDIR` like in `run.sh` and pushd/cd there prior to running the /usr/local/bin/openstack wrapper script.

Also I think we can save some time by moving the node delete out of the loop, since that supports passing multiple nodes (doesn't look like node maintenance set does though so we still need the loop)

Reply:
done